Napa.js build break on g++ 7.x

napajs: master branch on Apr 19
node: 9.11.1
g++: (GCC) 7.3.1 20180312
cmake: 3.10.3
cmake-js: 3.6.2

Last logs: 
```
../src/module/loader/module-loader-helpers.cpp:223:82: warning: ignoring return value of ‘v8::Maybe<bool> v8::Object::CreateDataProperty(v8::Local<v8::Context>, v8::Local<v8::Name>, v8::Local<v8::Value>)’, declared with attribute warn_unused_result [-Wunused-result]
             module->Get(v8_helpers::MakeV8String(isolate, "exports"))->ToObject());
                                                                                  ^
In file included from ../inc/napa/module/module-internal.h:13:0,
                 from ../src/module/loader/module-loader-helpers.h:6,
                 from ../src/module/loader/module-loader-helpers.cpp:4:
/home/darky/.cmake-js/node-x64/v9.11.1/include/node/v8.h:3123:37: note: declared here
   V8_WARN_UNUSED_RESULT Maybe<bool> CreateDataProperty(Local<Context> context,
                                     ^~~~~~~~~~~~~~~~~~
[35/81] Building CXX object src/CMakeFiles/napa.dir/providers/providers.cpp.o
ninja: build stopped: subcommand failed.
ERR! OMG Process terminated: 1
```

--- a/src/platform/process.cpp
+++ b/src/platform/process.cpp
@@ -21,6 +21,7 @@
 
 #include <sys/stat.h>
 
+#include <functional>
 #include <memory>
 #include <sstream>
 #include <string>

